the-chemical-society-of-japan.csl

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>The Chemical Society of Japan</title>
+    <id>http://www.zotero.org/styles/the-chemical-society-of-japan</id>
+    <link href="http://www.zotero.org/styles/the-chemical-society-of-japan" rel="self"/>
+    <link href="http://www.csj.jp/journals/styles/ref.html" rel="documentation"/>
+    <author>
+      <name>Shoji Takahashi</name>
+      <email>s.takahashi@elsevier.com</email>
+      <uri>http://www.mendeley.com/profiles/shoji-takahashi3/</uri>
+    </author>
+    <category citation-format="numeric"/>
+	<category field="chemistry"/>
+    <updated>2015-12-04T01:42:50+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author">
+    <names variable="author">
+      <name initialize-with=". "/>
+    </names>
+  </macro>
+  <macro name="container-title">
+    <choose>
+      <if type="article-journal" match="any">
+        <text variable="container-title" form="short" font-style="italic" suffix=" "/>
+      </if>
+      <else-if type="book" match="any">
+        <text variable="container-title" font-style="italic"/>
+      </else-if>
+      <else-if type="chapter" match="any">
+        <text term="in" suffix=" "/>
+        <text variable="container-title" font-style="italic"/>
+      </else-if>
+      <else>
+        <text variable="container-title" font-style="normal"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="edition">
+    <number suffix=" " variable="edition" form="ordinal"/>
+    <label plural="never" variable="edition" form="short"/>
+  </macro>
+  <macro name="event">
+    <text variable="event-place" suffix=", "/>
+    <date variable="event-date">
+      <date-part name="month" range-delimiter="" suffix=" "/>
+      <date-part name="day" range-delimiter="" suffix=", "/>
+      <date-part name="year" range-delimiter="" font-weight="bold"/>
+    </date>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="book article" match="any">
+        <text variable="title" font-style="italic"/>
+      </if>
+      <else>
+        <text variable="title"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="issued">
+    <date date-parts="year" form="numeric" variable="issued" font-weight="normal">
+      <date-part name="year" font-weight="bold"/>
+    </date>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <label form="verb-short" plural="never" suffix=" "/>
+      <name initialize="false" initialize-with=". "/>
+    </names>
+  </macro>
+  <macro name="locators">
+    <choose>
+      <if type="article-journal" match="any">
+        <number font-style="italic" variable="volume"/>
+      </if>
+      <else-if type="chapter" match="any">
```

Comment:
You'll actually want to place a group around lines 79-82, too. That way, the terms are only printed if you have the variable. Also, I'd prefer to use text rather than number at least for volume so that we're a bit more flexible with how volumes look (they're normally numbers, but I'm sure we can find number/letter combination in some contexts).